docs/src/administration/users.md

```diff
@@ -1,65 +1,53 @@
 ---
 title: "User administration"
 weight: 1
-sidebarTitle: "Users"
+sidebarTitle: "User administration"
 description: |
-  Every Platform.sh user has a role that controls their access and permission levels. Different roles allow different levels of access to your applications, environments and projects. You can manage how users interact with your project and environments at Platform.sh
+  Every Platform.sh user has a set of permissions. Depending on your role, you'll be able to access different levels of the application, environments and projects. If you want to add a user to a project or an environment type, the user has to create an account before they can contribute to the project.
 ---
 
 {{< description >}}
 
-Any user added to a project or an environment type on Platform.sh will need to [register for an account](https://auth.api.platform.sh/register) before they can contribute. If you need to delete your account at any time you can [transfer ownership](/administration/users.md#transfer-ownership) of your existing projects and then open a [support ticket](/development/troubleshoot.md#deleting-your-platformsh-account) to request your account deletion.  
+### User roles
 
-## User roles
+| User role    | Description |
+| ------------ |-------------|
+| Project owner                   | The organization where the project was created in the first place. There is only one and can create multiple project administrators. If you’re the owner of a project, and want to transfer it to a different user, first invite that user as a project administrator and then, submit a support ticket from your owner account to ask for the transfer. Transferring a project automatically transfers the upcoming subscription charges to the new owner. Once you transfer ownership, open a [support ticket](https://docs.platform.sh/development/troubleshoot.html) to request your account deletion. Note that transferring a project automatically transfers the upcoming subscription charges to the new owner. |
```

Comment:
I like the idea of making this a table. It has the potential to make this much more scanable. :+1: 

It's just that this is really long for tabular data. I don't think this works well as a table. I would include just a description here and separate out ideas about how to transfer ownership. That seems like a separate task and not a description of what an owner is.

Also, the owner isn't an organization, as is implied in the first sentence. There are places where that seems to be the case due to the way it was migrated, but the owner isn't the same thing as the organization.

Reply:
Thanks for the feedback.  We could definitely add a set of standardised words to our SG such as Project Admin or Admin or at least discuss which one to use. I am still not familiar with the Platform terminology so I always struggle picking the right one. I don't particularly like any existing option so I try to go to what I feel is the most logical choice. I wonder whether we can affect the product terminology decisions made by Maria by recommending things such as "Use Administrator o Project Administrator instead of Admin on the interface". It's probably an interesting discussion.

Reply:
We do use "Project Admin" in the user interface when inviting someone as such, so I'm not sure which scenario you're both describing. 

![image](https://user-images.githubusercontent.com/3729517/135818197-925c8b2a-fdb4-4c1e-8c1f-ba978c4879af.png)

Reply:
I meant in the overview of users in a given project.
![picture of users in Platform.sh UI with some Admins in project](https://user-images.githubusercontent.com/26428882/135818794-5bd812fd-b717-4b6f-afe2-9ad7d8140296.png)
It's just about consistency, at least within a single doc (everywhere in the UI and docs would be great, but not something that's done immediately).

Reply:
I have create a ticket from this comment so we can follow up the discussion separately. 
https://github.com/platformsh/platformsh-docs/issues/1918

Reply:
The other issue seems good for long-term planning. It would still be nice to have this doc consistent until that's settled. :slightly_smiling_face:

Reply:
So, I've settled for "Project admin" just to replicate the checkbox name.